tests/console/docker.pm

```diff
@@ -108,6 +108,16 @@ sub run {
     $output = script_output('docker container run tw:saved curl -I google.de');
     die("network is not working inside of the container tw:saved") unless ($output =~ m{Location: http://www\.google\.de/});
 
+    # docker run support --init parameter
+    assert_script_run 'docker run --rm --init opensuse/tumbleweed ps --no-headers -xo "pid args" | grep "1 /dev/init"';
+
+    # if man command is available
+    if (script_run('command -v man') == 0) {
+        # particular docker man pages are present
```

Comment:
I would remove this comment. The instructions under this line are self explanatory.